src/external/spglib.jl

```diff
@@ -1,40 +1,40 @@
-using PyCall
 # Routines for interaction with spglib
+const SPGLIB = spglib_jll.libsymspg
+
+function spglib_get_error_message()
+    error_code = ccall((:spg_get_error_code, SPGLIB), Cint, ())
+    return unsafe_string(ccall((:spg_get_error_message, SPGLIB), Cstring, (Cint,), error_code))
+end
 
 """
-Construct a tuple containing the lattice and the positions of the species
+Construct a tuple containing the positions of the species
 in the convention required to take the place of a `cell` datastructure used in spglib.
 """
-function spglib_cell_atommapping_(lattice, atoms)
-    lattice = Matrix{Float64}(lattice)  # spglib operates in double precision
-    n_attypes = isempty(atoms) ? 0 : sum(length(positions) for (type, positions) in atoms)
-    spg_numbers = Vector{Int}(undef, n_attypes)
-    spg_positions = Matrix{Float64}(undef, n_attypes, 3)
+function spglib_atommapping(atoms)
+    n_attypes = isempty(atoms) ? 0 : sum(length(positions) for (typ, positions) in atoms)
+    spg_numbers = Vector{Cint}(undef, n_attypes)
+    spg_positions = Matrix{Cdouble}(undef, 3, n_attypes)
 
     offset = 0
     nextnumber = 1
     atommapping = Dict{Int, Any}()
-    for (iatom, (type, positions)) in enumerate(atoms)
-        atommapping[nextnumber] = type
+    for (iatom, (typ, positions)) in enumerate(atoms)
+        atommapping[nextnumber] = typ
```

Comment:
Just out of curiousity: Any reason you removed the `e`?

Reply:
I always feel uneasy using built in things for variable names, and so does my syntax highligher :p. If you want I can put it back to type though

Reply:
I see, but where does `type` live in Julia? I did not find it in a fresh REPL at least.

Reply:
`type` isn't a keyword in julia 1.0 actually, only `abstract type` is

Reply:
oh that's true. I'll change it back